phpstan-baseline.neon

```diff
@@ -0,0 +1,337 @@
+parameters:
```

Comment:
great idea to add phpstan! and i agree with recording the baseline rather than trying to fix all in here.

we should open an issue to look into this file and see what we can fix / mark things as intentional.

Reply:
👍